### PR TITLE
rft!: mappings (suggestion)

### PR DESCRIPTION
**Pull request by ikerurda**
I propose using `<A-` for file management (add, rename, move, yank, delete)
and `<C-` for general file/folder-browsing.

| Insert / Normal | Action                                                           |
|-----------------|------------------------------------------------------------------|
| `<A-c>/c`       | Create file/folder at current path (trailing `/` creates folder) |
| `<A-r>/r`       | Rename multi-selected files/folders                              |
| `<A-m>/m`       | Move multi-selected files/folders to current path                |
| `<A-y>/y`       | Copy multi-selected files/folders to current path                |
| `<A-d>/d`       | Delete multi-selected files/folders                              |
| `<C-o>/o`       | Open file/folder with default system application                 |
| `<C-g>/g`       | Go to parent directory                                           |
| `<C-e>/e`       | Go to home directory                                             |
| `<C-w>/w`       | Go to current working directory (cwd)                            |
| `<C-t>/t`       | Change nvim's cwd to selected folder/file(parent)                |
| `<C-f>/f`       | Toggle between file and folder browser                           |
| `<C-h>/h`       | Toggle hidden files/folders                                      |
| `<C-a>/a`       | Toggle all entires ignoring `./` and `../`                       |

Thoughts?

### CODE REVIEW

**Comment by jamestrew** [comment]
I'm good with the proposed changes.

File: README.md [hunk 1]
```diff
@@ -72,19 +72,19 @@ vim.api.nvim_set_keymap(
...
-| `<C-y>/y`        | Copy (multi-selected) files or folders to cwd         |
-| `<C-d>/dd`       | Delete (multi-selected) files or folders              |
-| `<C-r>/r`        | Rename (multi-selected) files                         |
-| `<C-e>/e`        | Add File/Folder at cwd; trailing `/` creates folder   |
-| `--/m`           | Move multi-selected files to cwd                      |
-| `<C-h>/h`        | Toggle hidden files                                   |
+| `<A-c>/c`        | Add File/Folder at cwd; trailing `/` creates folder   |
+| `<A-r>/r`        | Rename (multi-selected) files                         |
+| `<A-m>/m`        | Move multi-selected files to cwd                      |
+| `<A-y>/y`        | Copy (multi-selected) files or folders to cwd         |
```
Comment (jamestrew) [inline README.md:78]: Shouldn't `cwd` be `path` (or maybe `current path`) for copy, move and create?

Comment (ikerurda) [reply README.md:78]: True, thanks!

**Comment by fdschmidt93** [comment]
In general, I personally don't care for the mapping so much myself -- I primarily assigned everything a default mapping for discoverability. Users should just remap actions as they want them to be mapped ("nudging").

Nevertheless, I like the underlying ideas of the PR with separating file management and other actions. However, this constitutes a larger breaking change which I would like to coincide with the full removal of file browser from telescope. Prior to that I would want to get

- a major docs PR (Gifs for common file management actions, code documentation for architecture, differences in multi selection, mappings, ...)
- opting in to tie `cwd` to `path` for folder browser because it's an entirely different approach to folder browsing that should be covered

in and then have like a proper 0.1 essentially (with removal of file browser) in which I'd want to include a major revisit of mappings then.

I wanted to do docs so long already, but little things keep popping up :sweat_smile: I'll move everything next week aside and prioritize that.

Does that work for you?

**Comment by ikerurda** [comment]
Yes, of course!  :)

**Comment by fdschmidt93** [comment]
As a heads up, I'll probably do this as part of all the things I want to get done today with a separate PR based off of latest master once other stuff is taking care of because this PR has conflicts, such that I can get all my breaking changes in within a brief time span. 

I'll close the PR later once I'm done.

Apologies and many thanks for getting the ball rolling here :)

**Comment by fdschmidt93** [comment]
Closing since superseded by #56

**Pull request closed without merging**